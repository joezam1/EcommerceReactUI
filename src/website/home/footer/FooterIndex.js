import React from 'react';
import NavigationFooter from './NavigationFooter.js';

export default function FooterIndex(){
    return(<div className="footer-section">
                <NavigationFooter/>
                <div className="footer-bottom-section">
                   <div className="footer-container-ul">
                        <div className="footer-item-li">
                            <div>
                                <ul className="footer-newsletter-ul">
                                    <li className="footer-newsletter-li">
                                        <span>Subscribe to our newsletter</span>
                                    </li>
                                    <li className="footer-newsletter-li">
                                        <input type="text" name="customer-email" className="newsletter-input" placeholder="your email here"/>
                                        <button type="button" className="newsletter-button" >Subscribe</button>
                                    </li>
                                </ul>
                                <li className="footer-content-li"><br/></li>
                                <li className="footer-content-li"><br/></li>
                                <span className="footer-title">Follow Us</span>
                                <ul className="footer-content-icons-ul">
                                    <li className="footer-content-li"> </li>    
                                   
                                    <li className="footer-content-li"><br/></li>
                                    <li className="footer-content-li">
                                        <img src="./assets/socialMedia/twitter_Logo_WhiteOnBlue.png" alt="twitter" className="twitter-icon"/>
                                    </li>                                    
                                    <li className="footer-content-li">
                                    <div className="facebook-icon-container">
                                        <img src="./assets/socialMedia/facebook-124010.png" alt="facebook" className="facebook-icon"/>
                                    </div>
                                      
                                    </li>
                                    <li className="footer-content-li">
                                        <img src="./assets/socialMedia/instagram-logo.png" alt="instagram" className="instagram-icon"/>
                                    </li>
                                    <li className="footer-content-li"></li>
                                </ul>
                               

                                <br/>                              
                            </div>
                        </div>      
                        <div className="footer-item-li">
                            <div>
                                <ul className="footer-content-ul">
                                    <li className="footer-content-li">
                                        <span className="footer-title">Company</span>
                                    </li>    
                                   
                                    <li className="footer-content-li"><br/></li>
                                    <li className="footer-content-li">About Us</li>                                    
                                    <li className="footer-content-li">Company History</li>
                                    <li className="footer-content-li">Location</li>
                                    <li className="footer-content-li">International Shipping FAQ</li>
                                    <li className="footer-content-li">Our Stores</li>
                                    <li className="footer-content-li">Unsubscribe to Newsletter</li>

                                </ul>
                            </div>
                        </div>
                        <div className="footer-item-li">
                           <div>
                                <ul className="footer-content-ul">
                                    <li className="footer-company-li">
                                        <span className="footer-title">Customer</span>
                                    </li>                                       
                                    <li className="footer-content-li"><br/></li>

                                    <li className="footer-content-li">My Account</li>                                    
                                    <li className="footer-content-li">Contact Us</li>
                                    <li className="footer-content-li">Our Services</li>
                                    <li className="footer-content-li">FAQs</li>
                                </ul>
                           </div>
                        </div>
                        <li className="footer-item-li">
                            <div>
                                <ul className="footer-content-ul">
                                    <li className="footer-content-li">
                                        <span className="footer-title">Our Blog</span>
                                    </li>    
                                    <li className="footer-content-li"><br/></li>

                                    <li className="footer-content-li">Articles</li>                                    
                                    <li className="footer-content-li">News</li>
                                    <li className="footer-content-li">Videos</li>
                                    <li className="footer-content-li"></li>
                                </ul>

                            </div>
                        </li>                      
                     </div>
                </div>
                
           </div>)
}