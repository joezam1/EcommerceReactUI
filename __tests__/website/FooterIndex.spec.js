import React from 'react';
import {render, cleanup, fireEvent, waitFor} from '@testing-library/react';
import FooterIndex from '../../src/website/home/footer/FooterIndex.js';


xdescribe('File FooterINdex.spec.js', function(){
    afterEach(cleanup);
    test('it should take a snapshot', function(){
        const container = render(<FooterIndex/>);
        expect(container.asFragment(<FooterIndex/>)).toMatchSnapshot();
    });
})