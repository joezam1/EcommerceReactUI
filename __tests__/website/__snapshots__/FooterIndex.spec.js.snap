// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File FooterINdex.spec.js it should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer-section"
  >
    <div
      class="navigation-footer"
    >
      <div
        class="footer-elements-ul"
      >
        <div
          class="footer-element-li"
          id="home"
        >
          Home
        </div>
        <div
          class="footer-element-li"
          id="about-us"
        >
          About Us
        </div>
        <div
          class="footer-element-li"
          id="terms-and-conditions"
        >
          Terms & Conditions
        </div>
        <div
          class="footer-element-li"
          id="shippings-and-returns"
        >
          Shippings & Returns
        </div>
        <div
          class="footer-element-li"
          id="privacy-policy"
        >
          Privacy Policy
        </div>
        <div
          class="footer-element-li"
          id="security"
        >
          Security
        </div>
      </div>
    </div>
    <div
      class="footer-bottom-section"
    >
      <div
        class="footer-container-ul"
      >
        <div
          class="footer-item-li"
        >
          <div>
            <ul
              class="footer-newsletter-ul"
            >
              <li
                class="footer-newsletter-li"
              >
                <span>
                  Subscribe to our newsletter
                </span>
              </li>
              <li
                class="footer-newsletter-li"
              >
                <input
                  class="newsletter-input"
                  name="customer-email"
                  placeholder="your email here"
                  type="text"
                />
                <button
                  class="newsletter-button"
                  type="button"
                >
                  Subscribe
                </button>
              </li>
            </ul>
            <li
              class="footer-content-li"
            >
              <br />
            </li>
            <li
              class="footer-content-li"
            >
              <br />
            </li>
            <span
              class="footer-title"
            >
              Follow Us
            </span>
            <ul
              class="footer-content-icons-ul"
            >
              <li
                class="footer-content-li"
              >
                 
              </li>
              <li
                class="footer-content-li"
              >
                <br />
              </li>
              <li
                class="footer-content-li"
              >
                <img
                  alt="twitter"
                  class="twitter-icon"
                  src="./assets/socialMedia/twitter_Logo_WhiteOnBlue.png"
                />
              </li>
              <li
                class="footer-content-li"
              >
                <div
                  class="facebook-icon-container"
                >
                  <img
                    alt="facebook"
                    class="facebook-icon"
                    src="./assets/socialMedia/facebook-124010.png"
                  />
                </div>
              </li>
              <li
                class="footer-content-li"
              >
                <img
                  alt="instagram"
                  class="instagram-icon"
                  src="./assets/socialMedia/instagram-logo.png"
                />
              </li>
              <li
                class="footer-content-li"
              />
            </ul>
            <br />
          </div>
        </div>
        <div
          class="footer-item-li"
        >
          <div>
            <ul
              class="footer-content-ul"
            >
              <li
                class="footer-content-li"
              >
                <span
                  class="footer-title"
                >
                  Company
                </span>
              </li>
              <li
                class="footer-content-li"
              >
                <br />
              </li>
              <li
                class="footer-content-li"
              >
                About Us
              </li>
              <li
                class="footer-content-li"
              >
                Company History
              </li>
              <li
                class="footer-content-li"
              >
                Location
              </li>
              <li
                class="footer-content-li"
              >
                International Shipping FAQ
              </li>
              <li
                class="footer-content-li"
              >
                Our Stores
              </li>
              <li
                class="footer-content-li"
              >
                Unsubscribe to Newsletter
              </li>
            </ul>
          </div>
        </div>
        <div
          class="footer-item-li"
        >
          <div>
            <ul
              class="footer-content-ul"
            >
              <li
                class="footer-company-li"
              >
                <span
                  class="footer-title"
                >
                  Customer
                </span>
              </li>
              <li
                class="footer-content-li"
              >
                <br />
              </li>
              <li
                class="footer-content-li"
              >
                My Account
              </li>
              <li
                class="footer-content-li"
              >
                Contact Us
              </li>
              <li
                class="footer-content-li"
              >
                Our Services
              </li>
              <li
                class="footer-content-li"
              >
                FAQs
              </li>
            </ul>
          </div>
        </div>
        <li
          class="footer-item-li"
        >
          <div>
            <ul
              class="footer-content-ul"
            >
              <li
                class="footer-content-li"
              >
                <span
                  class="footer-title"
                >
                  Our Blog
                </span>
              </li>
              <li
                class="footer-content-li"
              >
                <br />
              </li>
              <li
                class="footer-content-li"
              >
                Articles
              </li>
              <li
                class="footer-content-li"
              >
                News
              </li>
              <li
                class="footer-content-li"
              >
                Videos
              </li>
              <li
                class="footer-content-li"
              />
            </ul>
          </div>
        </li>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
